### PR TITLE
Update Twitter logo to new logo

### LINKED ISSUE #7781: Update the new Twitter logo
## Description
The Twitter logo needs to be replaced with the `X` logo in the [b.io](http://b.io/) footer and on the first blade of the home page based on [1].

[1] https://about.twitter.com/en/who-we-are/brand-toolkit

### PR DESCRIPTION
**Pull request by bhimeshagrawal**
## Purpose
Update the twitter logo in footer to twitterX logo

Fixes #7781

### CODE REVIEW

**Comment by sm1990** [comment]
@bhimeshagrawal ,

Thank you for your contribution. The issue #7781 requires you to update the twitter logo on the first blade of the home page as well.

File: components/common/footer/Footer.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function Footer() {
                       </li>
                       <li>
                         <a className={styles.footerLinkSM} href="https://twitter.com/ballerinalang" target="_blank" rel="noreferrer" title="Twitter">
-                          <Image src={`${prefix}/images/sm-icons/twitter-white.svg`} width={19} height={15} alt="Twitter" />
+                          <Image src={`${prefix}/images/sm-icons/twitter-x-white.svg`} width={19} height={19} alt="Twitter" />
```
Comment (sm1990) [inline components/common/footer/Footer.js:46]: Update the alt text as 'X' too.

Comment (bhimeshagrawal) [reply components/common/footer/Footer.js:46]: Sure

**Comment by dinushan20** [comment]
x-logo-white-color
Reference - https://about.twitter.com/en/who-we-are/brand-toolkit

**Comment by dinushan20** [comment]
@bhimeshagrawal Shall we use this icon file to align with our design. Ref - https://about.twitter.com/en/who-we-are/brand-toolkit. Thank you!
x-logo-white-color

**Comment by bhimeshagrawal** [comment]
> @bhimeshagrawal Shall we use this icon file to align with our design. Ref - https://about.twitter.com/en/who-we-are/brand-toolkit. Thank you! x-logo-white-color

this 

> @bhimeshagrawal Shall we use this icon file to align with our design. Ref - https://about.twitter.com/en/who-we-are/brand-toolkit. Thank you! x-logo-white-color

The logo I got from the twitter brand toolkit is in png format, can we use that or the file you have linked with this comment ?

**Comment by dinushan20** [comment]
> > @bhimeshagrawal Shall we use this icon file to align with our design.
> 
> The logo I got from the twitter brand toolkit is in png format, can we use that or the file you have linked with this comment ?

Yes, we can use both .png and .svg formats

**Comment by dinushan20** [comment]
> > > > @bhimeshagrawal Shall we use this icon file to align with our design.
> > > 
> > > 
> > > The logo I got from the twitter brand toolkit is in png format, can we use that or the file you have linked with this comment ?
> > 
> > 
> > Yes, we can use both .png and .svg formats
> 
> Ok, thanks for letting me know. I am updating the logo with the image you have linked with the comment in above conversations.

Super, Thank you very much!

**Review by sm1990 — APPROVED** [review]

**Pull request merged**